### ISSUE TITLE
Any active maintainers, or is repository abandoned?

### ISSUE THREAD

**Issue by tavour-frank** [react-native-push-notification/ios#350]
Not seeing a ton of movement in the open issues or pull requests for the repository, and not seeing any members in the react-native-push-notification organization. Has this project been abandoned?

@Naturalclar I see you as the approver for the most recent PRs back in 2021 - are you the current maintainer of the repository?

**Comment by Naturalclar** [comment]
@tavour-frank Hello 👋🏻  I'm still the current maintainer of the repository. I haven't been able to respond to the open issues as I am currently overwhelmed with work.

**Comment by tavour-frank** [comment]
@Naturalclar makes sense, are you looking for other maintainers/contributors? We make use of the project at my work so we're invested in maintaining and improving it as we encounter new issues or have new needs. We were debating taking a fork of the repo since it seemed stagnant but we'd prefer to contribute to the community solution

**Comment by surethink** [comment]
hope more contributors joining in, thank you all @Naturalclar @tavour-frank
